src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+$day-size: 100px;
+$transition-duration: 0.5s;
+$hover-transform: -20px;
+$hover-background: #ffbfcb;
+$gap: 1px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  align-items: center;
+  position: absolute;
```

Comment:
why do you need absolute positioning here? There are easier (and better) ways to center it
Using absolute position just to center element is not good practice, if there are other ways to do it